workflow/usage.rst

```diff
@@ -43,12 +43,13 @@ like this:
                     audit_trail:
                         enabled: true
                     marking_store:
-                        type: 'multiple_state' # or 'single_state'
+                        type: 'method'
                         arguments:
+                            - false
                             - 'currentPlace'
```

Comment:
1. The `arguments` key has been deprecated
1. the `false` / `true` to tell if it's a single / multiple marking store is now automatic
1. the `currentPlace` configuration now goes in a `property` key.

So I propose someting like:

```yaml
            marking_store:
                # Method will become the default value in Symfony 5.0
                type: method
                property: currentPlace
```

---

Don't forget to apply this comment on other configuration format